addons/l10n_cl/views/report_invoice.xml

```diff
@@ -154,14 +154,32 @@
 
         <t t-set="current_subtotal" t-value="current_subtotal + line.price_subtotal" position="before">
             <t t-set="l10n_cl_values" t-value="line._l10n_cl_prices_and_taxes()"/>
+            <t t-set="line_amounts" t-value="l10n_cl_values['line_amounts']"/>
         </t>
 
+        <xpath expr="//span[@t-field='line.price_unit']" position="before">
+            <t t-if="'second_currency' in line_amounts" t-set="line_second_currency_round" t-value="line_amounts['second_currency']['round_currency']"/>
+        </xpath>
+
         <xpath expr="//span[@t-field='line.price_unit']" position="attributes">
             <attribute name="t-field"></attribute>
-            <attribute name="t-esc">line.price_unit</attribute>
-            <attribute name="t-options">{"widget": "monetary", "display_currency": o.currency_id}</attribute>
+            <attribute t-if="not o.l10n_latam_document_type_id._is_doc_type_electronic_ticket()" name="t-esc">line_amounts['price_item_document']</attribute>
+            <attribute name="t-options">{"widget": "float", "precision": 2}</attribute>
```

Comment:
Hello, @jco-odoo so, if we put at it is now (widget float) it's ok, right? remember that the reason we have price_item_document in the dict, is because in some cases we use price_unit and in other cases we use the value we get from balance divided by the quantity (discounts, etc).

Reply:
Besides, as here we use CLP or even other currencies, monetary should not apply, so I think it's OK at it is now... @jco-odoo and we already removed the monetary...